### DISCUSSION TITLE
Icon Request: font-size-increase, font-size-decrease

### DISCUSSION THREAD

**Discussion by r0ml** [FortAwesome/Font-Awesome#21154] [Icon Requests]
Although there is a "font" icon as one of the text icons, it would be useful to have an "increase font size" and a "decrease font size" icon (perhaps a small A -> large A and  large A -> small A) to make it easy to scale text.  My particular use case is in making text annotations larger or smaller to match or offset the original text.

**Comment by u12206050** [comment]
+1

**Comment by researchware** [comment]
+1 Also very much looking for a A+ and A- icon pair for increasing/decreasing font size.

**Comment by leticia** [comment]
+1

**Comment by zhiwehu** [comment]
+1

**Comment by joshuarlowry** [comment]
+1

**Comment by dado4497** [comment]
+1

**Comment by fboucquez** [comment]
+1

**Comment by leogalmeida** [comment]
+1

**Comment by JoshuaDoshua** [comment]
This is an extremely common feature for working with site accessibility. While it can be achieved by putting a normal character in front of a +/-, it's difficult to make it look natural IMO.
What's the standard display for something like this? A+ A- or T+ T- ? I personally think the T, could be inferred as "text plus".

This icon should fall under text-editor and accessibility categories.

**Comment by sensibleworld** [comment]
We've added a text-size icon to Font Awesome Pro: https://fontawesome.com/icons/text-size?style=solid

**Comment by adircohe** [comment]
+1

**Comment by cassiuschen** [comment]
+1

**Comment by jaguileram** [comment]
+1

**Comment by SchreinerK** [comment]
Why this is closed? There are no diffent glyphs for font-size-increase, font-size-decrease, also not in pro.  I suggest 2 different glyphs for font resizing , 
- A with plus/minus 
- A with up/down arrow. 
- large A and small A combination / small A and large A combination  maybe with arrow
[https://prnt.sc/1tczpn0](https://prnt.sc/1tczpn0)
At the moment you have to put the symbols together in a cumbersome way.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
